source/compiler/qsc_frontend/src/resolve.rs

```diff
@@ -2366,11 +1982,14 @@ fn resolve_scope_locals(
                     });
                 }
             }
+            NameKind::Importable => {
+                // local variables are not importable
```

Comment:
Should this be a panic if it can never happen? Just to help a future refactor that might violate this invariant.

Reply:
Oh it can definitely happen. This block deals with local variables / type parameters. This code path is just saying "if you're looking for an importable, you're not going to find it among the local variables". But this function definitely _can_ get called with `NameKind::Importable`. In that case, `scope.item()` in the call below should return the item you're looking for.